app/services/CalculationService.scala

```diff
@@ -274,10 +274,11 @@ trait CalculationService {
 
   }
 
-  def determinePRRUsed (gain: Double, prrUsed: Option[Double]): Double = {
+  def determinePRRUsed (gain: Double, prrUsed: Option[Double], prr: String): Double = {
```

Comment:
Could the prr parameter be renamed to prrType or something to make it clearer? prrUsed would be more appropriate as prrValue too to avoid confusion with prrUsed in CalculatorController